### PR TITLE
Discord pollen post implemented

### LINKED ISSUE #266: Post "social-post" enabled pollens to discord channel

### PR DESCRIPTION
**Pull request by gokaykucuk**
Closes #266

### CODE REVIEW

File: app/src/backend/functions/social-post.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ export async function socialPost(platform, cid) {
     console.log("Social post disabled. Aborting...");
     return;
   }
-  const data = getPostData(ipfs, cid, platform === "twitter");
```
[the remark this answers is not in the document]
Comment (gokaykucuk) [reply app/src/backend/functions/social-post.js:35]: @voodoohop ah ok now i got what this does, I'll revert the change.

[the remark this answers is not in the document]
Comment (gokaykucuk) [reply app/src/backend/functions/social-post.js:35]: @voodoohop for discord we don't need to have anything special here as it's already building the message in another function. It's still not understandable what is being sent to getPostData here, so I'll create another constant for it.

[the remark this answers is not in the document]
Comment (voodoohop) [reply app/src/backend/functions/social-post.js:35]: Yeah sorry I should document these functions with generic names like getPostData much better. will try and add more comments

File: app/src/backend/functions/social-post.js [hunk 2]
```diff
@@ -32,7 +33,9 @@ export async function socialPost(platform, cid) {
...
+
+  const shortenPost = platform === "twitter" ? true : false
```
Comment (voodoohop) [inline app/src/backend/functions/social-post.js:37]: tiny nitpick:
`const shortenPost = platform === "twitter";` its a bit more concise

**Review by voodoohop — APPROVED** [review]

**Comment by voodoohop** [comment]
We could move these files out of the `backend/functions` folder. that one was actually meant to be used for netlify functions but we're not using them

**Pull request merged**